_posts/2017/11/2017-11-08-online-meetings.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+authors: ["Belinda Weaver"]
+title: "Running effective online meetings with Zoom (or Google Hangouts)"
+date: 2017-11-08
+time: "00:00:00"
+category: ["Online meetings", "Zoom"]
+---
+ 
+Online meetings are a fact of life for most of us in the Carpentries. So how do we make them as effective as we can?
+
+This post came about after the Twitter discussion generated by this [tweet from Titus Brown](https://twitter.com/ctitusbrown/status/926165114948001792).
+
+Here are some points I have thought of. Feel free to comment.
+
+#### Before the meeting:
+
```

Comment:
Familiarize yourself with the agenda for the meeting. Are there things you need to figure out beforehand? Are there things you are wondering about? If there is no agenda, ask the Chair to send one out.